post_office/mail.py

```diff
@@ -334,3 +337,25 @@ def send(email):
     )
 
     return len(sent_emails), num_failed, num_requeued
+
+
+def send_all_in_queue(lockfile=default_lockfile, processes=1, log_level=None):
```

Comment:
What trips me up about this function name is the `all` part because it implies that everything that's queued will be sent, but in practice the ones that are scheduled for future delivery won't.

But I also can't think of a better name for this at this moment. I thought about `send_until_done()`, `enter_send_loop()` but these are also weird.

If we can't think of a better name, I think we should just rename this to `send_queued_mail` since this is also what the management command is called so there's consistency there at least.

Reply:
I renamed that function to `send_queued_mail_until_done`. This reflects exactly what it does, and since that function is intended to be used internally, I don't have any problems with such a long name.
I strongly would discourage to reuse the name `send_queued_mail`. This can cause hefty confusion.